### PR TITLE
refactor(presets): migrate ai preset to AFFiNE

### PR DESCRIPTION
**Pull request by L-Sun**
## TL;DR
This PR migrates AI **presets** to AFFiNE.
Related PR: https://github.com/toeverything/AFFiNE/pull/7219

## What You Should Know
- After the migration, the new AI preset folder on the AFFiNE side will be:
`packages/frontend/core/src/blocksuite/presets/ai`
- Some AI-related code is not in the preset folder and is heavily coupled with the basic functionality of BlockSuite. This issue may lead to another PR to refactor these parts:
  - `packages/blocks/src/root-block/widgets/ai-panel`
This file is coupled with the `format-bar`.
  - `packages/blocks/src/rootblock/edgeless/controllers/tools/copilot-tool.ts`
This file is coupled with the `EdgelessToolManager`.

  - `packages/blocks/src/root-block/widgets/edgeless-copilot`
  - `packages/blocks/src/root-block/widgets/edgeless-copilot-panel`
  - `packages/blocks/src/_common/components/ai-item`
  - `packages/blocks/src/_common/icons/ai.ts`

### CODE REVIEW

**Comment by L-Sun** [comment]
* **#7322** <a href="https://app.graphite.dev/github/pr/toeverything/blocksuite/7322?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `master`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @L-Sun and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Review by doodlewind — APPROVED** [review]
